src/core/account/account-pixie.js

```diff
@@ -223,17 +223,17 @@ const accountPixie: TamePixie<AccountProps> = combinePixies({
       let dirty = lastActiveWalletIds !== activeWalletIds
       lastActiveWalletIds = activeWalletIds
 
-      let lastOut = {}
+      let lastOut: { [walletId: string]: EdgeCurrencyWallet } = {}
       if (accountOutput != null && accountOutput.currencyWallets != null) {
         lastOut = accountOutput.currencyWallets
       }
 
-      const out = {}
+      const out: { [walletId: string]: EdgeCurrencyWallet } = {}
       const { wallets } = input.props.output.currency
       for (const walletId of activeWalletIds) {
-        const api = wallets?.[walletId]?.walletApi
+        const api = wallets[walletId]?.walletApi
         if (api !== lastOut[walletId]) dirty = true
-        out[walletId] = api
+        if (api != null) out[walletId] = api
```

Comment:
Wouldn't it be safer if you added this after 234?
`if (api == null) continue`

Reply:
No, that would be incorrect. We want to do the diff in every case, in case the wallet gets archived & then restored.